.github/workflows/cd.yaml

```diff
@@ -0,0 +1,119 @@
+name: cd
+
+on:
+  push:
+    tags:
+      - "v[0-9]+.[0-9]+.[0-9]+"
+  workflow_run:
+    workflows:
+      - ci
+    types:
+      - completed
+    branches:
+      - master
+
+env:
+  TERM: xterm
```

Comment:
is there a requirement for this?

Reply:
Yep . Makefile needs that because it has some erroneous errors